apps/backoffice-v2/src/domains/storage/fetchers.ts

```diff
@@ -37,14 +36,20 @@ export const fetchFileInfoById = async (fileId: string) => {
 
   return handleZodError(error, fileInfo);
 };
-export const fetchFileById = async (fileId: string) => {
+export const fetchFileById = async ({
+  fileId,
+  withSignedUrl = true,
+}: {
+  fileId: string;
+  withSignedUrl?: boolean;
+}) => {
   const fileInfo = await fetchFileInfoById(fileId);
 
-  if (fileInfo.fileNameInBucket && env.VITE_FETCH_SIGNED_URL) {
+  if (withSignedUrl && fileInfo?.fileNameInBucket) {
```

Comment:
Moved env condition higher as part of the `withSignedUrl` argument so the conditions for signed URL are clearer and easily overridable.